### PR TITLE
Update resources after sdk generator refactor

### PR DESCRIPTION
**Pull request by agualis**
Note that we are manually keeping the deprecated transactions `update` function until we replace it by `StorefrontPostPayment` in storefront-payment-form.

### CODE REVIEW

**Review by p3yman — DISMISSED** [review]

File: src/resources/storefront/invoices-resource.js [hunk 1]
```diff
@@ -4,20 +4,19 @@
...
-        const params = {limit, offset, filter, q, sort, expand};
-        return apiHandler.getAll(`invoices`, params);
-      },
-      get({id}) {
-        return apiHandler.get(`invoices/${id}`);
-      },
-    };
-  }
+  return {
+    getAll({
```
Comment (ashkarpetin) [inline src/resources/storefront/invoices-resource.js:8]: `expand` is lost

Comment (ashkarpetin) [reply src/resources/storefront/invoices-resource.js:8]: It is actively used

Comment (ashkarpetin) [reply src/resources/storefront/invoices-resource.js:8]: Nevermind, it is Storefront

File: src/resources/storefront/transactions-resource.js [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
-
-        get({id} = {}) {
-            return apiHandler.get(`transactions/${id}`);
-        },
+/**
+* This file was auto-generated by rebilly-sdk-generator.
+* Do not make direct changes to this file.
+*/
 
-        update({id, data} = {}) {
```
Comment (ashkarpetin) [inline src/resources/storefront/transactions-resource.js:18]: I wonder why patch removed?

Comment (ashkarpetin) [reply src/resources/storefront/transactions-resource.js:18]: StorefrontPatchTransaction is used by storefront-payment-form

**Review by ashkarpetin — CHANGES_REQUESTED** [review]
PatchTransaction should remain.

File: src/resources/storefront/transactions-resource.js (hunk 2 above)
Comment (agualis) [reply src/resources/storefront/transactions-resource.js:18]: We were talking about that with Arif. This patch is deprecated and we should use `StorefrontPostPayment` instead. But 
I will manually add this update until do that migration.

**Review by ashkarpetin — APPROVED** [review]

**Comment by ashkarpetin** [comment]
Do we need to bump version?

**Pull request merged**